pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Validate AsyncAPI document with Studio"
+description: This tutorial will teach you how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+## Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+You will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
+
+## Background context
+An AsyncAPI document is a file that defines and annotates the different components of a specific Event-Driven API. The format of the file must be JSON or YAML. You can use this document to generate both documentation and code.
+
+The AsyncAPI Studio tool allows you to develop an AsyncAPI document, validate it, preview it, convert it to the latest version, and visualize event flows.
+
+If you are following tutorial without doing the previous tutorial [Create AsyncAPI document]() then quickly generate an AsyncAPI file using `asyncapi new --example=tutorial.yml --no-tty`. You can see that the documentation is now rendered and you don't see any errors which means your file is valid. Now let's experiment with an invalid file to see how errors are displayed and how to make that file valid again.
+
+## Copy invalid AsyncAPI document
+Let's pretend we have an invalid AsyncAPI document.
+
+1. Open [Studio](https://studio.asyncapi.com/).
+
+2. Copy and paste the below invalid AsyncAPI document:
+
+```yaml
+{`asyncapi: '1.0.0'
```

Comment:
hmm, where did you take this from, it refers to ancient asyncapi 1.0.0 😄 
also please remove 
```
{`
```
and 
```
`}
```